### DISCUSSION TITLE
How do I provide provisioning profiles for app-extensions?

### DISCUSSION THREAD

**Discussion by sag333ar** [sparkfabrik/ios-build-action#3] [Q&A] [answered]
This github workflow supports adding provisioning profile only for main app. I've multiple app-extensions like widget and siri-extensions. I would like to provide provisioning profiles for the same. How do I achieve that with this github work flow?

**Comment by edodusi** [comment, marked as answer]
Hi @sag333ar, sorry this feature is not implemented yet, but I can point you to https://github.com/yukiarrr/ios-build-action because I know they now support multiple provisioning files.
